voicebox_pytorch/voicebox_pytorch.py

```diff
@@ -295,7 +295,10 @@ def __init__(
             kernel_size = conv_pos_embed_kernel_size,
             groups = conv_pos_embed_groups
         )
-
+        # if we are using mel spec with 80 channels, we need to set attn_channels to 80
+        # dim_in assuming we have spec with 80 channels
+        self.aligner = Aligner(dim_in = 80, dim_hidden = dim_phoneme_emb, attn_channels = 80)
```

Comment:
we could probably have the `dim_in` and `attn_channels` as hyperparameters, perhaps passed in as `aligner_kwargs: dict = dict(dim_in = 80, attn_channels = 80)` as default